### DISCUSSION TITLE
Adding Correspondence field to already created fields

### DISCUSSION THREAD

**Discussion by ulvidamirli** [directus/directus#12543] [Feature Requests]
I want to be able to add the correspondence field not only while creating but also after creating the field for collection.

Problem: 
Most of the time I mirror my database to Directus which gives no chance to have correspondence fields on my collections.

Solution:
To have a correspondence field option on update field page.

**Comment by azrikahar** [comment]
Assuming the correspondence field in question is referring to fields like `user_created`, `date_created` etc, then there are existing discussions on this such as #8458 and #4686 👍

**Reply by ulvidamirli** [reply]
I don't mean only these fields but also correspondence fields mainly for M2O and M2M relations

**Reply by azrikahar** [reply]
Ohh thanks for clarifying. That'd be nice indeed

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
